Features/Context/ContentActions.php

```diff
@@ -25,10 +48,11 @@ public function iSeeCopiedNotification($name, $destiny)
      */
     public function moveInto($name, $destiny)
     {
-        $this->onFullView($name);
-        $this->clickActionBar('Move');
-        $this->selectFromUniversalDiscovery("eZ Platform/$destiny");
-        $this->confirmSelection();
+        $this->contentEditContext->onFullView($name);
+        $this->waitWhileLoading();
```

Comment:
seems that the `onFullView()` needs to wait afterwards.
If so, adding it there instead would allow removing all waits after using it.